src/qt/rpcconsole.cpp

```diff
@@ -474,9 +474,9 @@ RPCConsole::RPCConsole(interfaces::Node& node, const PlatformStyle *_platformSty
     const QString list{"<ul><li>" + Join(CONNECTION_TYPE_DOC, QString("</li><li>")) + "</li></ul>"};
     ui->peerConnectionTypeLabel->setToolTip(ui->peerConnectionTypeLabel->toolTip().arg(list));
     const QString hb_list{"<ul><li>\""
-        + tr("To") + "\" – " + tr("we selected the peer for high bandwidth relay") + "</li><li>\""
-        + tr("From") + "\" – " + tr("the peer selected us for high bandwidth relay") + "</li><li>\""
-        + tr("No") + "\" – " + tr("no high bandwidth relay selected") + "</li></ul>"};
+        + tr_to   + "\" – " + tr("we selected the peer for high bandwidth relay") + "</li><li>\""
+        + tr_from + "\" – " + tr("the peer selected us for high bandwidth relay") + "</li><li>\""
+        + tr_no   + "\" – " + tr("no high bandwidth relay selected")              + "</li></ul>"};
```

Comment:
Does [`clang-format-diff.py`](https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy) agree with extra spaces?

Reply:
Dropped (a bit of a shame though, seems less readable) and did the same for other touched lines.